lib/mustache/generator.rb

```diff
@@ -118,12 +120,22 @@ def t.tokens(src=@source)
             p.compile(src)
           end
           t.render(ctx.dup)
-        else
-          # Shortcut when passed non-array
-          v = [v] unless v.is_a?(Array) || v.is_a?(Mustache::Enumerable) || defined?(Enumerator) && v.is_a?(Enumerator)
+        compiled
+      end
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [`end` at 124, 6 is not aligned with `if` at 110, 22.](https://www.codacy.com/app/rokusu/mustache/file/2024324680/issues/source?bid=2903613&fileBranchId=3121922#l124)